.github/dependabot.yml

```diff
@@ -19,5 +19,8 @@ updates:
 - package-ecosystem: gomod
   directory: /
   schedule:
-    interval: daily
+    interval: weekly
+    day: monday
```

Comment:
I had thought we had decided on Sunday so they would be ready first thing Monday morning. If we want to avoid weekend activity then how about 5AM Central on Monday.

Reply:
We kinda left it open in discussion but I'm happy to move it to the suggested time which is as early as anyone rightly ought to be working in US ET.